### ISSUE TITLE
Unexpected accuracy loss with 0.9.0

### ISSUE THREAD

**Issue by opensorceror** [mozilla/DeepSpeech#3404]
- **Have I written custom code (as opposed to running examples on an unmodified clone of the repository)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10 Pro, v1909
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: Your builds I believe
- **TensorFlow version (use command below)**: v2.3.0-6-g23ad988fcd
- **Python version**: 3.7.6
- **Bazel version (if compiling from source)**: N/A
- **GCC/Compiler version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: 

```sh
deepspeech --model deepspeech-0.9.0-models.pbmm --scorer deepspeech-0.9.0-models.scorer --audio nis.wav
```

```sh
deepspeech --model deepspeech-0.9.0-models.pbmm --scorer deepspeech-0.9.0-models.scorer --audio google_scam.wav
```

I'm seeing a strange regression from 0.8.2 to 0.9.0. Not sure if it is coincidental and specific to my test data. Attached are two audio files. [audio.zip](https://github.com/mozilla/DeepSpeech/files/5485267/audio.zip)

With DeepSpeech 0.8.2, here are the results of each:
1. `nis.wav`:
```sh
$ deepspeech --model deepspeech-0.8.2-models.pbmm --scorer deepspeech-0.8.2-models.scorer --audio nis.wav
Loading model from file deepspeech-0.8.2-models.pbmm
TensorFlow: v2.3.0-6-g23ad988fcd
DeepSpeech: v0.9.0-0-g2d04fbe0
2020-11-03 22:03:06.182282: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Loaded model in 0.0313s.
Loading scorer from files deepspeech-0.8.2-models.scorer
Loaded scorer in 0.0192s.
Warning: original sample rate (8000) is different than 16000hz. Resampling might produce erratic speech recognition.
Running inference.
the number you have dialed is not in service please check the number and try your call again thank you the number you have dialed is not in service please check the number and try your call again thank you
Inference took 10.701s for 21.740s audio file.
```

2. `google_scam.wav`:
```sh
$ deepspeech --model deepspeech-0.8.2-models.pbmm --scorer deepspeech-0.8.2-models.scorer --audio google_scam.wav
Loading model from file deepspeech-0.8.2-models.pbmm
TensorFlow: v2.3.0-6-g23ad988fcd
DeepSpeech: v0.9.0-0-g2d04fbe0
2020-11-03 22:02:43.001753: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Loaded model in 0.0314s.
Loading scorer from files deepspeech-0.8.2-models.scorer
Loaded scorer in 0.0147s.
Running inference.
this is an important message do not hang up the phone your google business or thing is it risk of deploying and correctly we tried numerous times to contact you by telephone so we can check the status of your lifting this will only take a few minutes of your time that one so we can quickly check the status of your google listing to ensure its that there if you're golitsin or deleted your business will not be so in coughing customers to think you're no longer in for 
business plus one now catechists of your glisten if you were the business one now that's to be removed from this last
Inference took 15.041s for 30.276s audio file.
```

The transcription accuracy for `nis.wav` is perfect (100%), while that for `google_scam.wav` is pretty high. However, with 0.9.0, the accuracy drops significantly (the inference seems faster though). Here are the results with 0.9.0:

1. `nis.wav`:
```sh
$ deepspeech --model deepspeech-0.9.0-models.pbmm --scorer deepspeech-0.9.0-models.scorer --audio nis.wav
Loading model from file deepspeech-0.9.0-models.pbmm
TensorFlow: v2.3.0-6-g23ad988fcd
DeepSpeech: v0.9.0-0-g2d04fbe0
2020-11-03 22:10:05.557952: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Loaded model in 0.0274s.
Loading scorer from files deepspeech-0.9.0-models.scorer
Loaded scorer in 0.0155s.
Warning: original sample rate (8000) is different than 16000hz. Resampling might produce erratic speech recognition.
Running inference.
the number you have dialed is not in service please check the number and try your call again thank you outnumber you have interested any
Inference took 10.165s for 21.740s audio file.
```

2. `google_scam.wav`:
```sh
$ deepspeech --model deepspeech-0.9.0-models.pbmm --scorer deepspeech-0.9.0-models.scorer --audio google_scam.wav
Loading model from file deepspeech-0.9.0-models.pbmm
TensorFlow: v2.3.0-6-g23ad988fcd
DeepSpeech: v0.9.0-0-g2d04fbe0
2020-11-03 22:09:27.709886: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Loaded model in 0.028s.
Loading scorer from files deepspeech-0.9.0-models.scorer
Loaded scorer in 0.0177s.
Running inference.
this the important message do not hang up the phone your google business a thing if it was of displaying and correctly we tried numerous times to contact you by telephone on we can cheapest of your lifting the leontines of your time that one so we can quickly check the status of your gillingwater you're localised or delatour business will not be so nominating you no monotonousness buonconte the status of your listeners one now as to to be removed from the west      
Inference took 13.935s for 30.276s audio file.
```

Am I the only one experiencing this?

**Comment by rhamnett** [comment]
You are not specifying the 0.9.0 best lm_alpha and lm_beta.

**Comment by helllynx** [comment]
Faced with same issue. My recognition quality is even lower, for example, the word `hello` is recognized as `hlo`. I try to amplify input wav file, use good mic and etc. Nothing changed.

**Comment by reuben** [comment]
It's not completely unexpected. The 0.9.0 model is the first one trained with data augmentation, but due to [recent changes at Mozilla](https://blog.mozilla.org/blog/2020/08/11/changing-world-changing-mozilla/) we haven't been able to fully adjust all hyperparameters after the introduction of the new augmentation ones. This is reflected for example on the slightly worse WER on LibriSpeech-test-clean. Our hope is that it at least represents an improvement in noisy conditions, since that's the main focus of the data augmentation. In any case, the 0.8.2 model should be compatible with the 0.9.0 clients, so you can also stick to the previous model/checkpoints. Another possibility is doing some fine tuning of the 0.9.0 checkpoint, if you have some data. Maybe that would cover the accuracy gap for you. If you do that I'd be very interested in hearing what results you got.

**Comment by reuben** [comment]
I'll close this in favor of using [Discourse](https://discourse.mozilla.org/c/deep-speech) for this topic, as it is not a bug or feature request.
